feeds/nabsa.github.com.dmfr.json

```diff
@@ -5,7 +5,7 @@
       "id": "f-aar~bike~konya~tr~gbfs",
       "spec": "gbfs",
       "urls": {
-        "gbfs_auto_discovery": "https://app.linkalock.com/api/gbfs/D2uRLGw8vBa2FJxa8/…"
```

Comment:
If you'd like to submit fixes for GBFS feeds, please open a PR against https://github.com/MobilityData/gbfs/blob/master/systems.csv

That file is automatically synced into the Transitland Atlas repo.